elements/cv_scaler.cc

```diff
@@ -196,11 +203,19 @@ void CvScaler::Read(
     note_ += 0.1f * interval;
   }
 
-  float modulation = pot_lp_[POT_RESONATOR_FM_ATTENUVERTER] * 49.5f *
+  // manually read the values of the exciter blow meta cv and input
+  // attenuverter to tranform the value of the FM attenuverter pot
+  float pot_fm = pot_lp_[POT_RESONATOR_FM_ATTENUVERTER];
+  float cv_fm = calibration_settings_.offset[CV_ADC_EXCITER_BLOW_META] -
```

Comment:
you didn't mention utilizing the calibration cv offset, but sense every other use of cv includes it, I decided to use it as well.

I followed `calibration_settings_.offset` until the main function's ui.DoEvents() call.  It seems that Element's takes in the calibration CVs every time the "switch" is pressed (which I assume is the button you press which starts the excitation.)  I'm not really understanding what that is doing.